Samples/resource-owner-password-credential/Angular2/src/newoidc/wwwroot/app/authorize/authoriza-service.ts

```diff
@@ -0,0 +1,71 @@
+﻿import {Injectable}     from '@angular/core';
+import {Http, Response, Headers, RequestOptions} from '@angular/http';
+import {logModel, registerModel, extprovider,token,Regresult} from './authorize-component'
+import {Configuration} from '../app.constants'
+import {Observable} from 'rxjs/Rx';
+import 'rxjs/add/operator/map';
+@Injectable()
+export class authervice {
```

Comment:
current status : ropc working (logout with my custom fucntion)
question:
1. how do i make logout on connect/logout its throwing error
2. with external login i can get either id-token or access-token but its not authorizing thowing error.
i will remove that "Angular2" folder once its all working.